Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponse/ProofpointThreatResponse.py

```diff
@@ -721,7 +721,7 @@ def main():
     elif command == 'proofpoint-tr-get-incident':
         get_incident_command()
 
-    elif command == 'proofpoint-tr-add-comment-to-incident':
+    elif command == 'proofpoint-tr-update-incident-comment':
```

Comment:
are you sure it's the right re-name?

Reply:
That's what was written in the issue. I saw in addition that there is no implementation for the command proofpoint-tr-update-incident-comment that appears in the documentation commands list

Reply:
Let's ask whoever wrote this integration what should be the correct name, or look at the API.
Does it add an incident comment, or updates one?

Reply:
@yuvalbenshalom @liorblob Which one of you wrote that integration? 
Do you think that proofpoint-tr-update-incident-comment is the correct command? it was missing before .